docs/dev/release.rst

```diff
@@ -188,20 +183,78 @@ You can create a clean checkout as follows (requires
      Other Changes and Additions
      ---------------------------
 
-#. Commit your changes of the, uh, change log::
+     A.C.1 (unreleased)
+     ==================
 
-     git add CHANGES.rst
-     git commit -m "Back to development: A.B.dev"
+     Bug Fixes
```

Comment:
Do we also want to pre-populate the Viz sub-sections here?

Reply:
Good call, I'll add them.